### PR TITLE
Cheatsheet templates added

### PR DESCRIPTION
**Pull request by Sloff**
This adds templates to all the cheatsheets. Some examples:

anniversary
coffee script
drush
game of thrones houses
german
gnu screen
harry potter spells
irc
lgbt
morse code
nato
perldoc
poker
svn
tennis

### CODE REVIEW

**Comment by moollaza** [comment]
Woah! @Sloff thanks a lot!!

**Comment by abeyang** [comment]
Awesome. :+1:

**Comment by zekiel** [comment]
dannnnng nice work @Sloff

**Comment by Sloff** [comment]
Haha, thanks all!

On Tue, Aug 18, 2015 at 12:11 AM, Zac Pappis < notifications@github.com [notifications@github.com] > wrote:
dannnnng nice work @Sloff [https://github.com/Sloff]

—
Reply to this email directly or view it on GitHub
[https://github.com/duckduckgo/zeroclickinfo-goodies/pull/1408#issuecomment-131979432] .[https://github.com/notifications/beacon/AG_ZejmQavm7aBLFO07rWc2PSq2fp9Otks5oolONgaJpZM4FtJTU.gif]

**Comment by russellholt** [comment]
nice :zap:

**Comment by tagawa** [comment]
Wow, that definitely deserves :cake:

**Comment by jeet09** [comment]
nice! looks good :)

File: share/goodie/cheat_sheets/json/ed.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
         "sourceName": "EdCheatSheet",
         "sourceUrl": "https://www.freebsd.org/cgi/man.cgi?ed%281%29"
     },
+    "template_type": "keyboard",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/ed.json]: I think we need to switch this one to `terminal`

**Comment by moollaza** [comment]
Tested and they all look great!

**Comment by anothernewphone** [comment]
Hi @Sloff, we love your contributions and would love to send you some swag!  Can you email us at merch @ duckduckgo .com?  Thanks!

**Pull request merged**